src/wasm-binary.h

```diff
@@ -1059,6 +1059,18 @@ enum MemoryAccess {
 
 enum MemoryFlags { HasMaximum = 1 << 0, IsShared = 1 << 1, Is64 = 1 << 2 };
 
+enum ElementFlags {
+  // Bit 0: 0 means active segment, 1 means passive or declarative
+  IsInactive = 1 << 0,
+  // Bit 1 if passive or declarative: 0 means passive, 1 means declarative
+  IsDeclarative = 1 << 1,
```

Comment:
Can we say what it means to be "declarative" in the comment?

Reply:
Added a comment above `SegmentFlag` enum explaining all three types of segments.